source/overloading.tex

```diff
@@ -3486,24 +3480,23 @@
 \indextext{overloading!unary operator}
 
 \pnum
-A prefix unary operator shall be implemented by a
-non-static member function\iref{class.mfct} with no parameters or a
-non-member function with one parameter.
+For a prefix unary operator \tcode{@},
+an operator function \tcode{\keyword{operator}@} shall be
+a non-static member function\iref{class.mfct} with no parameters or
+a non-member function with one parameter.
```

Comment:
I wonder if phrasing these as "is" rather than "shall" would work better:

> A prefix unary operator function for a *unary-operator* `@` is an operator function `operator@` that is either a non-static member function with no parameters or a non-member function with one parameter.

(Then the new "shall" requirement gives us the requirement that `operator+` effectively has either one or two parameters, and so on.)

Reply:
Good idea.

Reply:
Fixed.